.travis.yml

```diff
@@ -101,30 +101,34 @@ matrix:
     # Build Linux wheels.
     - os: linux
       env:
-        - LINUX_WHEELS=1
+        - LINUX_WHEELS=1 MULTIPLATFORM_JARS=1
         - PYTHONWARNINGS=ignore
         - RAY_INSTALL_JAVA=1
       install:
-        - . ./ci/travis/ci.sh init RAY_CI_LINUX_WHEELS_AFFECTED
+        - . ./ci/travis/ci.sh init RAY_CI_LINUX_WHEELS_AFFECTED,RAY_CI_JAVA_AFFECTED,RAY_CI_STREAMING_JAVA_AFFECTED
       before_script:
         - . ./ci/travis/ci.sh build
       script:
         - . ./ci/travis/ci.sh test_wheels
+        - bash ./java/build-jar-multiplatform.sh linux
+        - bash ./java/build-jar-multiplatform.sh multiplatform
```

Comment:
Better put the `multiplatform` part in the second job. Because the second one is likely to be slower.

Reply:
The benefits of putting theose together is we can exit early if build wheel failed. If we put it in a seperate job, it may need to wait 2 hous because it doesn;t whether dependent job failed or just not finished, occupy resources which can be used by other job